README.md

```diff
@@ -137,24 +158,6 @@ A value of `max_rolledover_indices=20` on a three node bare-metal cluster with t
 
 ends up consuming a constant of `407GiB` per node.
 
-It is recommended to store any track parameters in a json file and pass them to Rally using `--track-params=./params-file.json`. Example content:
```

Comment:
This is a rather specific example related to this challenge (which we use for our long running benchmarks), so for the sake of having it as a reference I suggest we keep it, but reworded as:

> The following is an example of passing the parameters used by this challenge with the help of a track parameters file:

Reply:
Makes sense.  I can re-word it to keep the example in this section.

Reply:
Let me know if this looks acceptable.

Reply:
I think it's ok I'd personally say instead of `of parameters that could be sent to the challenge.` -> `configurable parameters for this challenge` but it's a matter of taste.

Reply:
Done I liked your way of putting it.